app/src/main/res/drawable/ic_history_search.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
did I miss this being used somewhere?

Reply:
Yes, it's being used when creating the search engine.  I'll comment at the location.

Reply:
As agreed with @gabrielluong issue created as a follow up to see if we can support themes in search engine icons https://github.com/mozilla-mobile/android-components/issues/12187